### ISSUE TITLE
Discrepancy in historical data

### ISSUE THREAD

**Issue by mattttt** [raintank/grafana#512]
We took a snapshot a couple months ago and it included a very distinct pattern from Singapore:
https://app.raintank.io/dashboard/snapshot/AEiNkio7vPufq0W3eUbxOdnxOBIe8z1p

But when I took a snapshot today, Singapore does not appear in any of the graphs:
https://snapshot.raintank.io/dashboard/snapshot/HM4QPak2elOnx1lMIVqYSa9Pb4ZNVyEm

As such, the uptime stats mildly differ as well, At the time of taking the second snapshot today, the Singapore collector was **Disabled**, but was not disabled at the time of the incident.

**Comment by woodsaj** [comment]
This is expected.  The metricDefinition index (which stores the full set of metrics that exist) has had significant changes over the last few months. 

When making the changes we did not perform any migration and instead just cleared the index and let it get rebuilt as new metrics were received.  
As the singapore collector has never been enabled since the schema was changed, there are no entries in elasticsearch for singapore data.
